lib/verify.go

```diff
@@ -130,8 +131,17 @@ func VerifyChain(certs []*x509.Certificate, ocspStaple []byte, dnsName, caPath s
 		result.Error = fmt.Sprintf("%s", err)
 		return result
 	}
+	// expectedName could be a hostname or could be a SPIFFE ID (spiffe://...)
+	// x509 package doesn't support verifying SPIFFE IDs. We thus have to do a bit more work here.
+	spiffeIDExpected := strings.HasPrefix(expectedName, "spiffe://")
+	var expectedDnsName string
+	if spiffeIDExpected {
+		expectedDnsName = ""
```

Comment:
This case can be omitted, since it's a no-op right now?

```
var expectedDNSName string
if !spiffeIDExpected {
  expectedDnsName = expectedName
}
```

Reply:
agreed